website/www/site/content/en/documentation/sdks/python-custom-multi-language-pipelines-guide.md

```diff
@@ -270,6 +270,23 @@ inspect.signature(MyTransform)
 
 This metadata is generated directly from the provider's implementation. The class documentation is generated from the [optional **description** method](#additional-metadata). The signature information is generated from the `@SchemaFieldDescription` annotations in the [configuration object](#implement-a-configuration).
 
+### Using Beam native Java SchemaTransforms
+If there's an existing Beam native Java SchemaTransform you'd like to use, and you know which expansion service module it's in, you can connect to it using `BeamJarExpansionService`:
+
+```python
+from apache_beam.transforms.external_transform_provider import ExternalTransformProvider
+from apache_beam.transforms.external import BeamJarExpansionService
+
+identifier = "beam:schematransform:org.apache.beam:bigquery_fileloads:v1"
```

Comment:
do we have the list of these identifiers? If so, we could add a link here.

Reply:
No comprehensive list yet unfortunately

Reply:
https://github.com/apache/beam/blob/3132bffbbdbc1c9eb3adcb0d226789f30f5efc43/sdks/standard_expansion_services.yaml#L57 is this something helpful here?

Reply:
It's not a complete list

Reply:
We should probably store all of these in proto like here: 
https://github.com/apache/beam/blob/1cf9755f040ef4aa76d36374f9ffd884114aba8b/model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/external_transforms.proto#L62-L80

and maybe use it to generate documentation on the Beam website